src/lib.rs

```diff
@@ -458,6 +458,7 @@ pub mod string {
     /// # Errors
     ///
     /// Can produce an [`Error`](crate::Error) when the formatting fails.
+    #[allow(clippy::missing_panics_doc)]
```

Comment:
This comes from the below line:

```rust
Ok(String::from_utf8(buf).expect("formatted string should be valid UTF-8"))
```

which is a "cannot happen"-style panic, which we opt to use over unsafe.